src/main/steps/applicant1/finalising-your-application/content.ts

```diff
@@ -63,10 +159,17 @@ const languages = {
 };
 
 export const generateContent: TranslationFn = content => {
+  const { userCase } = content;
   const translations = languages[content.language](content);
+  const AwaitingFinalOrderState = State.AwaitingFinalOrder.includes(userCase.state as State);
```

Comment:
why not make it equals as its not an array and you can just do that for both.

Reply:
also maybe change the name to isAwaitingFinalOrderState or something like that

Reply:
Hi @velimd I've changed the names to isAwaitingFinalOrderState and isAwaitingJointFinalOrderState Am not too sure what you mean by making it equals, as if I remove the .includes(userCase.state as State) to State.AwaitingFinalOrder it doesn't display the right content.

Reply:
const isAwaitingFinalOrderState = userCase.state === State.AwaitingFinalOrder;
strict comparson like we do in other places.

Reply:
Thanks @velimd I updated it.